src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -248,7 +248,8 @@ const ContextMenuActions: ContextMenuAction[] = [
         shouldShow: (type, reportAction, isArchivedRoom, betas, menuTarget, isChronosReport, reportID, isPinnedChat, isUnreadChat) =>
             type === CONST.CONTEXT_MENU_TYPES.REPORT_ACTION || (type === CONST.CONTEXT_MENU_TYPES.REPORT && !isUnreadChat),
         onPress: (closePopover, {reportAction, reportID}) => {
-            Report.markCommentAsUnread(reportID, reportAction?.created);
+            const originalReportID = ReportUtils.getOriginalReportID(reportID, reportAction) ?? '';
```

Comment:
What does "original" mean in this context?

Reply:
Sorry I'm not sure what you're trying to say can you try to rephrase it?

Reply:
Oh I think I have seen that referred to as a "parentReportID" can we use that term here?

Reply:
Ok fair enough. Its probably outside the scope of these changes then.